modules/nextMillenniumBidAdapter.js

```diff
@@ -328,6 +333,30 @@ function getWurl({auctionId, requestId}) {
   return wurlMap[key] && wurlMap[key].wurl;
 }
 
+function eventHandler(eventName) {
+  const _eventHandler = getEventHandler(eventName);
+  if (eventName == CONSTANTS.EVENTS.BID_TIMEOUT) {
+    return bids => {
+      if (!Array.isArray(bids)) return
+
+      for (let bid of bids) {
+        _eventHandler(bid);
+      };
+    }
+  };
+
+  return _eventHandler;
+}
+
+function getEventHandler(eventName) {
+  return bid => {
+    const bidder = bid.bidder || bid.bidderCode;
+    if (bidder != BIDDER_CODE) return;
+
+    triggerPixel(`${REPORT_ENDPOINT}?event=${eventName}&bidder=${bidder}&source=pbjs`);
+  }
+}
```

Comment:
@patmmccann I corrected the code and now all tests pass. I don't know how to test this new functionality, all the added code does is set an event handler for the bidRequested, bidResponse, noBid and bidTimeout events. Internally, it generates a url and sends data to the server via the `utils.triggerPixel` function.
If you still insist on tests, I will try to add them.

Reply:
@patmmccann Can you please tell me what you think about it

Reply:
There are no unit tests on this pr, we're hesitant to assign a reviewer before the pr is complete

Reply:
Seeking some feedback from others on how to test, will get back to you